pkg/controller/mutators/core/controller_test.go

```diff
@@ -16,18 +16,22 @@ limitations under the License.
 package core
```

Comment:
Should this test file be renamed?  It was previously `controller_core.go` and `controller_test.go`, but `controller_core.go` has been replaced by `reconciler.go`

Reply:
I like controller_test since it's the test that actually instantiates the controller, whereas reconciler_test doesn't.